platform/os_windows.go

```diff
@@ -118,3 +137,48 @@ func KillProcessByName(processName string) {
 	cmd := fmt.Sprintf("taskkill /IM %v /F", processName)
 	ExecuteCommand(cmd)
 }
+
+// executePowershellCommand executes powershell command
+func executePowershellCommand(command string) (string, error) {
+	ps, err := exec.LookPath("powershell.exe")
+	if err != nil {
+		return "", fmt.Errorf("Failed to find powershell executable")
+	}
+
+	cmd := exec.Command(ps, command)
+	var stdout bytes.Buffer
+	var stderr bytes.Buffer
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	cmd.Run()
+
+	return strings.TrimSpace(stdout.String()), nil
+}
+
+// SetSdnRemoteArpMacAddress sets the regkey for SDNRemoteArpMacAddress needed for multitenancy
+func SetSdnRemoteArpMacAddress() error {
+	if sdnRemoteArpMacAddressSet == false {
```

Comment:
!sdnRemoteArpMacAddressSet or do setRemoteArpMacAddress = false initially then check setRemoteArpMacAddress

Reply:
I don't understand this comment.